content/ch-algorithms/teleportation.ipynb

```diff
@@ -22580,1081 +22580,66 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Exception in Tkinter callback\n",
```

Comment:
Is this a bug in qiskit?

Reply:
Apologies for that. It is just the error log when using matplotlib with jupyter. I never specified that the plots be `svg`. Is this alright or shall I execute the code with `svg` plots enabled?

Reply:
yes please, svg is preferred as they render better on the site

Reply:
Hey Frank! I ran the cells with svg enabled. Also, I added `job_monitor` for job status updates.

Reply:
Thank you!